app/components/case-studies-components/CaseStudiesOtherCaseStudies.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import Image from "next/image";
+import Link from "next/link";
+
+import { CaseStudyHeaderProps } from "@/utils/interfaces";
+
+const truncateString = (str: string, num: number): string => {
+  if (str.length <= num) {
+    return str;
+  }
+  return str.slice(0, num) + "...";
+};
```

Comment:
This function, `truncateString`, could be moved into a utility component.

Reply:
I'm importing this now. Way cleaner